pyqtgraph/parametertree/parameterTypes/action.py

```diff
@@ -1,8 +1,10 @@
 from ...Qt import QtCore, QtWidgets, QtGui
+from ...functions import disconnect
 from ..Parameter import Parameter
 from ..ParameterItem import ParameterItem
 
 
+
```

Comment:
Very minor: It would be best to remove this blank space to maintain pep-8 guidelines